### PR TITLE
Home Work 3 JS Conditions and Loops

### PR DESCRIPTION
**Pull request by AndreyTests**

### CODE REVIEW

File: js-conditions-and-loops/conditions.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//Using "if else"
+const averageGrade = 85;
+
+if (averageGrade <= 59){
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js]: just as advice, use if (averageGrade < 60). It is less confusing. But your code is valid

File: js-conditions-and-loops/conditions.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    console.log("Незадовільно");
+
+}else if (averageGrade>60&&averageGrade<=70){
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js]: please add spaces for conditions. It is hard to read
`averageGrade > 60 && averageGrade <= 70`

Comment (sttaran) [reply js-conditions-and-loops/conditions.js]: your previous condition is `<= 59` and this one `>60`. In the end value 60 is not handled at all

Comment (sttaran) [reply js-conditions-and-loops/conditions.js]: it should be >=60

Comment (sttaran) [reply js-conditions-and-loops/conditions.js]: same for next conditions

File: js-conditions-and-loops/conditions.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    console.log("Дуже добре")
+
+}else if (averageGrade>91&&averageGrade<=100){
+    console.log("Відмінно")}
+
+// Using "switch case"
+const averageGrade1 = 95;
+
+switch (true) {
+    case averageGrade1 <= 59:
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js]: same issues as in previous loop

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js-conditions-and-loops/conditions.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+//Using "if else"
+const averageGrade = 85;
+
+if (averageGrade < 60){
+    console.log("Незадовільно");
+
+}else if (averageGrade >= 60 && averageGrade <= 70){
+    console.log ("Задовільно")
+
+}else if (averageGrade > 71 && averageGrade <= 80){
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js]: value 71 is not handled
image

Same for next conditions

File: js-conditions-and-loops/conditions.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    console.log("Добре")
+
+}else if (averageGrade > 81 && averageGrade <=9 0){
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js]: remove extra space averageGrade <= 90

File: js-conditions-and-loops/conditions.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    console.log("Відмінно")}
+
+// Using "switch case"
+const averageGrade1 = 95;
+
+switch (true) {
+    case averageGrade1 < 60:
+        console.log("Незадовільно")
+        break
+    case averageGrade1 >= 60 && averageGrade1 <= 70:
```
Comment (sttaran) [inline js-conditions-and-loops/conditions.js:26]: border values (71, 81, 91) are not handled

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js-conditions-and-loops/conditions.js (hunk 1 above)
Comment (AndreyTests) [reply js-conditions-and-loops/conditions.js]: fixed

File: js-conditions-and-loops/conditions.js (hunk 2 above)
Comment (AndreyTests) [reply js-conditions-and-loops/conditions.js]: fixed

File: js-conditions-and-loops/conditions.js (hunk 4 above)
Comment (AndreyTests) [reply js-conditions-and-loops/conditions.js]: fixed

**Pull request merged**